server/controller/user.js

```diff
@@ -0,0 +1,116 @@
+import db from '../models/index';
+import { validateSignup, validateLogin, comparePwd } from '../validators/validator.js';
+import { sendValidationError, serverError, sendSuccess, sendFail } from '../reply/reply';
+import { generateToken } from '../auth/auth';
+const Users = db.Users;
+const log = console.log;
```

Comment:
why do you have to do this? Can you send a response message instead?

Reply:
I guess u are asking about this '+const log = console.log'
is to log out an error to the console